apps/next/app/server/util/constants.ts

```diff
@@ -0,0 +1 @@
+export const DIR_FOR_CLIENT_DEPLOYMENTS = "ups";
```

Comment:
I don't think this should be an exported / shared thing--it's temp space specific to the /up route and it would be asking for trouble to share the state of that directory with other logic in the app